docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,13 +2,174 @@
 [role="xpack"]
 == SIEM UI
 
+IMPORTANT: This document is a work in progress. 
+It covers features scheduled for beta release in 7.2.
+
 beta[]
 
-After you have security monitoring <<install-siem,up and
-running>> and data is streaming to {es}, use the {siem-ui} in {kib} to monitor
-and identify security problems in real time.
+After you have security monitoring <<install-siem,up and running>> and data is
+streaming to {es}, check out the {siem-ui} in {kib}. You can monitor, identify,
+track, and notate potential security issues in real time, or look back
+historically to build a case.
+
+The {siem-app} is powerful, flexible, and highly interactive. The UI elements
+are designed to be discoverable, clickable, draggable and dropable, expandable
```

Comment:
Not 100% on this one, but what about "draggable and dropable" => "drag & drop-able"?

Or perhaps pop out of the "-able" and say "supports drag & drop to the Timeline" or something to that effect.